python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -586,6 +587,10 @@ def resolve_updateTime(self, graphene_info: ResolveInfo):
         run_record = self._get_run_record(graphene_info.context.instance)
         return datetime_as_float(run_record.update_timestamp)
 
+    def resolve_creationTime(self, graphene_info: ResolveInfo):
+        run_record = self._get_run_record(graphene_info.context.instance)
```

Comment:
looking at `__init__` it seems we are always constructed with record now, seems like we can remove this `_get_run_record` jonx

Reply:
good point https://github.com/dagster-io/dagster/pull/22805